08.ls_object/file_detail.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'etc'
+
+class FileDetail
+  COLUMN_WIDTH = 30
+
+  FILE_PERMISSION = {
+    '0' => '---',
+    '1' => '--x',
+    '2' => '-w-',
+    '3' => '-wx',
+    '4' => 'r--',
+    '5' => 'r-x',
+    '6' => 'rw-',
+    '7' => 'rwx'
+  }.freeze
+
+  def initialize(directory, name)
```

Comment:
```suggestion
  def initialize(file_path)
```

のように最初から`/foo/bar/baz.txt`のような文字列を受け取るようにした方が、引数が減ってシンプルになるかもしれません